chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java

```diff
@@ -493,7 +493,8 @@ public void check() {
     if (!isV2Open()) {
       List<String> cpList = getCheckpointList();
       if (cpList != null && cpList.size() != 0) {
-        logger.error("checkpoint check failed, can't convert checkpoint from v2 to v1");
+        logger.error("checkpoint check failed, can't convert checkpoint from v2 to v1," +
```

Comment:
may be not converted by user,  the database can be downloaded from somewhere

Reply:
The reason should be the database not match to the config file, so just print " the checkpoint version of database not match your config file ,please ..." ?